Sources/WalletConnectSign/Engine/Common/DeletePairingService.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import WalletConnectKMS
+import WalletConnectUtils
+import WalletConnectPairing
+
+class DeletePairingService {
```

Comment:
Can we move it in WalletConnectPairing package?

Reply:
oh, I see. Whey using different NetworkInteractors..

Reply:
yeah exactly, even the protocol method is the same